### ISSUE TITLE
504 Gateaway Time-out

### ISSUE THREAD

**Issue by HDsports** [elinw/AssetFix#6]
I would use the assetfix for Joomla 3, because after Migration from 1.5 to 3.x my site performance was very bad. When i save an article in a category with thousand existing articles i must wait 2 minutes, because there running lot of ordering processes. i am sure, i have these problem, because i have corrupt assets table.

But when i run the assetfix.php i wait many minutes and than comes the error 504 Gateaway Time-Out

Have you any idea, what i can do?

**Comment by elinw** [comment]
I think we figured this out right? Closing but let me know if I should reopen.

**Comment by HDsports** [comment]
Hey Elin,

i had give up. Dont found a way to solve this problem.

Best regards, tom

Grünbergstrasse 29/7

A-1120 Wien

+43 660 234 02 85

 http://www.hdsports.at/ www.HDsports.at //  mailto:office@HDsports.at office@HDsports.at

Österreichs Laufsport- und Leichtathletikplattform: Laufkalender / Laufranking Austria – die größte Laufdatenbank Europas / Ergebnisdienst / News aus Laufsport und Leichtathletik / Gesundheits- und Trainingstipps / Run It Videos & Fotos

Schenk uns ein Like:  https://www.facebook.com/hdsports.at www.facebook.com/hdsports.at 

Von: elinw [mailto:notifications@github.com] 
Gesendet: Sonntag, 09. August 2015 18:05
An: elinw/AssetFix
Cc: HDsports
Betreff: Re: [AssetFix] 504 Gateaway Time-out (#6)

I think we figured this out right? Closing but let me know if I should reopen.

—
Reply to this email directly or view it on GitHub https://github.com/elinw/AssetFix/issues/6#issuecomment-129206248 .

---

Diese E-Mail ist frei von Viren und Malware, denn der avast! Antivirus Schutz ist aktiv.
https://www.avast.com/antivirus
